### PR TITLE
chore(package.json): clear up intermidiate build artifact after build

### PR DESCRIPTION
**Pull request by kwonoj**
⚠️ Targeting master ⚠️

- [x] Non-breaking change

**Description:**
This PR makes force clear intermediate build artifact `es6` used for internal doc / umd package building only, to no longer included in actual 

**Related issue (if exists):**
#2177

### CODE REVIEW

**Comment by kwonoj** [comment]
I'm feeling bulid script might need to be simplified (and remove some of confusions if anyone read scripts).. but that's other story later.

File: package.json [hunk 1]
```diff
@@ -57,11 +57,11 @@
     "build_es6": "npm-run-all clean_dist_es6 copy_src_es6 compile_module_es6",
     "build_es6_for_docs": "npm-run-all clean_dist_es6 copy_src_es6 compile_dist_es6_for_docs",
     "build_closure_core": "node ./tools/make-closure-core.js",
-    "build_global": "npm-run-all clean_dist_global build_es6 && mkdirp ./dist/global && node ./tools/make-umd-bundle.js && npm-run-all build_closure_core",
+    "build_global": "npm-run-all clean_dist_global build_es6 && mkdirp ./dist/global && node ./tools/make-umd-bundle.js && npm-run-all build_closure_core clean_dist_es6",
     "build_perf": "webdriver-manager update && npm-run-all build_cjs build_global perf",
     "build_test": "shx rm -rf ./dist/ && npm-run-all build_cjs clean_spec build_spec test_mocha",
     "build_cover": "shx rm -rf ./dist/ && npm-run-all build_cjs build_spec cover",
-    "build_docs": "npm-run-all build_global build_es6_for_docs build_cjs clean_spec build_spec tests2png decision_tree_widget && esdoc -c esdoc.json",
+    "build_docs": "npm-run-all build_global build_es6_for_docs build_cjs clean_spec build_spec decision_tree_widget && esdoc -c esdoc.json && npm-run-all clean_dist_es6",
```
Comment (jayphelps) [inline package.json]: should tests2png have been removed? It's no longer used in any other script and I think it is indeed needed as part of the doc building process?

Comment (kwonoj) [inline package.json]: OH NO 😱 my bad, I was removing it to test script without png generation tools setup.

**Comment by kwonoj** [comment]
Updated PR corrects my error.

**Comment by jayphelps** [comment]
Seems good, but I want @blesh to confirm as well.

**Comment by kwonoj** [comment]
@jayphelps I agree.

**Comment by blesh** [comment]
LGTM

**Pull request merged**